### ISSUE TITLE
Copy paste cheat

### ISSUE THREAD

**Issue by SopaXorzTaker** [SopaXorzTaker/TPT-Alchemy#3]
Currently, when opening an external save and copy-pasting something from it, the pasted particles wouldn't count, but the new particles would still be counted. I think we should set `ignoreElementAcquistion` when the snippet was copied from an external save.

**Comment by SopaXorzTaker** [comment]
@jacob1, is this issue resolved?

**Comment by jacob1** [comment]
No, there complete anti-cheat protection right now (so this bug is still there)

**Comment by SopaXorzTaker** [comment]
Reopening, since it is still incredibly annoying.

**Comment by SopaXorzTaker** [comment]
That commit made playing the game a bad experience.
